pkgconfig/oeenclave-clang.pc

```diff
@@ -5,13 +5,17 @@ prefix=${pcfiledir}/../..
 exec_prefix=${prefix}
 includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+
 mbedtlslibs=@ENCLAVE_MBEDTLS_CRYPTO_LIBS@
 openssllibs=@ENCLAVE_OPENSSL_CRYPTO_LIBS@
+openssl_3libs=@ENCLAVE_OPENSSL_3_CRYPTO_LIBS@
 openssl_symcrypt_fipslibs=@ENCLAVE_OPENSSL_SYMCRYPT_FIPS_CRYPTO_LIBS@
 
+openssl_3flags=@ENCLAVE_OPENSSL_3_INCLUDES@
+
 Name: Open Enclave
 Description: Open Enclave clang flags and libraries for building enclaves.
 Version: @OE_VERSION@
 Requires:
```

Comment:
I think you need to walk me through this. What is the value of @ENCLAVE_OPENSSL_3_CRYPTO_LIBS@?

Reply:
That value is defined in pkgconfig/CMakeLists.txt.
@ENCLAVE_OPENSSL_3_CRYPTO_LIBS@ is "-loecryptoopenssl_3 -lopensslssl_3 -lopensslcrypto_3". These are build arguments to add libraries containing the OpenSSL 3.0 source code.
@ENCLAVE_OPENSSL_3_INCLUDES@ is "-I\${includedir}/openenclave/3rdparty/openssl_3". This is a build argument to add an include directory that contains OpenSSL 3.0 header files.